ui/src/views/OpenVPNTun.vue

```diff
@@ -1493,9 +1496,7 @@ export default {
         },
         {
           label: this.$i18n.t("openvpn_tun.vpn_network"),
-          field: function(rowObj) {
-            return rowObj.Network ? rowObj.Network : rowObj.LocalPeer;
-          },
+          field: "Network",
           filterable: true,
           sortFn: function(a, b, col, rowX, rowY) {
             a = a.indexOf("-") > -1 ? a.split("-")[0] : a;
```

Comment:
I think you can only fix the `sortFn` function to avoid `indexOf of null` error by check the value like this:
```suggestion
            a = a && a.indexOf("-") > -1 ? a.split("-")[0] : a;
```

have you tried?

Reply:
Yes, trying to protect by checking the value exist does not work, this is the next error

```
VM113 chunk-vendors.fc1ee862.js:17 TypeError: Cannot read property 'split' of undefined
    at sortFn (VM114 app.279de4d5.js:1)
    at VM113 chunk-vendors.fc1ee862.js:125
    at Array.sort (<anonymous>)
    at Array.<anonymous> (VM113 chunk-vendors.fc1ee862.js:17)
    at VM113 chunk-vendors.fc1ee862.js:125
    at Array.forEach (<anonymous>)
    at o.processedRows (VM113 chunk-vendors.fc1ee862.js:125)
    at nr.get (VM113 chunk-vendors.fc1ee862.js:17)
    at nr.evaluate (VM113 chunk-vendors.fc1ee862.js:17)
    at o.processedRows (VM113 chunk-vendors.fc1ee862.js:17)
at @ VM113 chunk-vendors.fc1ee862.js:17
rt @ VM113 chunk-vendors.fc1ee862.js:17
tt @ VM113 chunk-vendors.fc1ee862.js:17
nr.get @ VM113 chunk-vendors.fc1ee862.js:17
nr.run @ VM113 chunk-vendors.fc1ee862.js:17
qn @ VM113 chunk-vendors.fc1ee862.js:17
(anonymous) @ VM113 chunk-vendors.fc1ee862.js:17
lt @ VM113 chunk-vendors.fc1ee862.js:17
Promise.then (async)
it @ VM113 chunk-vendors.fc1ee862.js:17
_t @ VM113 chunk-vendors.fc1ee862.js:17
er @ VM113 chunk-vendors.fc1ee862.js:17
nr.update @ VM113 chunk-vendors.fc1ee862.js:17
pe.notify @ VM113 chunk-vendors.fc1ee862.js:17
set @ VM113 chunk-vendors.fc1ee862.js:17
rr.set @ VM113 chunk-vendors.fc1ee862.js:17
sort @ VM113 chunk-vendors.fc1ee862.js:125
click @ VM113 chunk-vendors.fc1ee862.js:125
nt @ VM113 chunk-vendors.fc1ee862.js:17
n @ VM113 chunk-vendors.fc1ee862.js:17
Ka.i._wrapper @ VM113 chunk-vendors.fc1ee862.js:17
VM113 chunk-vendors.fc1ee862.js:17 TypeError: Cannot read property 'split' of undefined
    at sortFn (VM114 app.279de4d5.js:1)
    at VM113 chunk-vendors.fc1ee862.js:125
    at Array.sort (<anonymous>)
    at Array.<anonymous> (VM113 chunk-vendors.fc1ee862.js:17)
    at VM113 chunk-vendors.fc1ee862.js:125
    at Array.forEach (<anonymous>)
    at o.processedRows (VM113 chunk-vendors.fc1ee862.js:125)
    at nr.get (VM113 chunk-vendors.fc1ee862.js:17)
    at nr.evaluate (VM113 chunk-vendors.fc1ee862.js:17)
    at o.processedRows (VM113 chunk-vendors.fc1ee862.js:17)
at @ VM113 chunk-vendors.fc1ee862.js:17
rt @ VM113 chunk-vendors.fc1ee862.js:17
tt @ VM113 chunk-vendors.fc1ee862.js:17
nr.run @ VM113 chunk-vendors.fc1ee862.js:17
qn @ VM113 chunk-vendors.fc1ee862.js:17
(anonymous) @ VM113 chunk-vendors.fc1ee862.js:17
lt @ VM113 chunk-vendors.fc1ee862.js:17
Promise.then (async)
it @ VM113 chunk-vendors.fc1ee862.js:17
_t @ VM113 chunk-vendors.fc1ee862.js:17
er @ VM113 chunk-vendors.fc1ee862.js:17
nr.update @ VM113 chunk-vendors.fc1ee862.js:17
pe.notify @ VM113 chunk-vendors.fc1ee862.js:17
set @ VM113 chunk-vendors.fc1ee862.js:17
rr.set @ VM113 chunk-vendors.fc1ee862.js:17
sort @ VM113 chunk-vendors.fc1ee862.js:125
click @ VM113 chunk-vendors.fc1ee862.js:125
nt @ VM113 chunk-vendors.fc1ee862.js:17
n @ VM113 chunk-vendors.fc1ee862.js:17
Ka.i._wrapper @ VM113 chunk-vendors.fc1ee862.js:17
VM113 chunk-vendors.fc1ee862.js:17 TypeError: Cannot read property 'split' of undefined
    at sortFn (VM114 app.279de4d5.js:1)
    at VM113 chunk-vendors.fc1ee862.js:125
    at Array.sort (<anonymous>)
    at Array.<anonymous> (VM113 chunk-vendors.fc1ee862.js:17)
    at VM113 chunk-vendors.fc1ee862.js:125
    at Array.forEach (<anonymous>)
    at o.processedRows (VM113 chunk-vendors.fc1ee862.js:125)
    at nr.get (VM113 chunk-vendors.fc1ee862.js:17)
    at nr.evaluate (VM113 chunk-vendors.fc1ee862.js:17)
    at o.processedRows (VM113 chunk-vendors.fc1ee862.js:17)
    ```
    
  then if I try to protect the next split

Reply:
I have a fix @edospadoni I commit it